packages/data-row-permissions/src/store/actions.js

```diff
@@ -0,0 +1,80 @@
+import api from '@molgenis/molgenis-api-client'
+
+export const actions = {
+  getAllRoles ({ commit }) {
+    api.get('/api/data/sys_sec_Role').then((response) => {
```

Comment:
Handle errors for all api calls, i.e. What if the server goes down or the session has timed out?